v2/devdoc/logging.md

```diff
@@ -238,6 +238,12 @@ void a(LOG_CONTEXT_HANDLE log_context, const char* correlation_id)
 
 Note: When logging with local contexts, the context passed to `LOGGER_LOG` has to be passed using the address of the context (&).
 
+If too many properties are passed when defining local contexts, the error shall be indicated by calling a log internal error report function.
+
+### LOG_ABORT_ON_ERROR
+
+The internal error report function shall abort the execution if `LOG_ABORT_ON_ERROR` is defined so that such error cases can easily be noticed during development.
```

Comment:
Interesting. Errr... so when I write unittests and I check parameters for non-NULL values,... and I actually pass a NULL value... hoping that the function under test will return some non-zero value... 

... the execution will be aborted?

Reply:
Nope. It would be aborted if too many properties are shoved in a log context and they would not fit in the stack arrays that we use for stack contexts.

The idea being that during development we should get an indication that we're adding too many properties (or generally too much data to a logging context) so that we can go increase the sizes for example.